simtools/layout/layout_array.py

```diff
@@ -376,7 +376,9 @@ def _get_corsika_sphere_center(self, tel_name):
 
         """
 
-        if names.get_telescope_type(tel_name) is not None:
+        if (names.get_telescope_type(tel_name) is not None
```

Comment:
When would `get_telescope_type` return `None`? Looking at the code, I am not sure it ever would.

Reply:
True, fixed!